src/components/select/select-list/select-list.component.js

```diff
@@ -318,17 +335,7 @@ const SelectList = React.forwardRef(
       };
     }, [assignListWidth]);
 
-    useLayoutEffect(() => {
-      let newHeight;
-
-      newHeight = listRef.current.clientHeight;
-
-      if (listActionButtonRef.current) {
-        newHeight += listActionButtonRef.current.parentElement.clientHeight;
-      }
-
-      setListHeight(`${newHeight}px`);
-    }, [children]);
+    useLayoutEffect(updateListHeight, [children]);
```

Comment:
I might be wrong but I think we don't need `useLayoutEffect` anymore here.

Reply:
OK, I'll give it a go. I never gave it much thought, I just assumed as it was already using `useLayoutEffect` there would be a good reason (particularly as it's setting the size of DOM elements).